packages/react-input/src/stories/InputAppearance.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { Label } from '@fluentui/react-label';
+import { useId } from '@fluentui/react-utilities';
+import { makeStyles } from '@fluentui/react-make-styles';
+import { Input } from '../index';
+
+const useStyles = makeStyles({
+  root: {
+    '& label': { display: 'block', paddingBottom: '2px' },
+    '& label:not(:first-child)': { paddingTop: '20px' },
+  },
+});
+
+export const Appearance = () => {
+  const outlineId = useId('input-outline');
+  const underlineId = useId('input-underline');
+  const filledLighterId = useId('input-filledLighter');
+  const filledDarkerId = useId('input-filledDarker');
+  const styles = useStyles();
+
+  return (
+    <div className={styles.root}>
+      <Label htmlFor={outlineId}>Outline (default)</Label>
+      <Input appearance="outline" id={outlineId} placeholder="placeholder" />
```

Comment:
For this and many of the others (appearance, disabled, inline, size), I'd like to have either a placeholder or a defaultValue to demonstrate at a glance how it will look.